RxStarscreamTests/RxStarscreamTests.swift

```diff
@@ -0,0 +1,159 @@
+//
+//  RxStarscreamTests.swift
+//  RxStarscreamTests
+//
+//  Created by Cezary Kopacz on 25/03/2017.
+//  Copyright © 2017 Guy Kahlon. All rights reserved.
+//
+
+import XCTest
+import RxSwift
+import RxTest
+import Starscream
+import RxStarscream
+
+extension WebSocketEvent: Equatable { }
+
+public func ==(lhs: WebSocketEvent, rhs: WebSocketEvent) -> Bool {
+    switch (lhs, rhs) {
+    case (.connected, .connected):
+        return true
+    case (.disconnected(_), .disconnected(_)):
+        return true
+    case (.message(_), .message(_)):
```

Comment:
is there any reason to skip checking for message content equality?
the same below with `.data` and above with `.disconnected`